MapboxNavigation/NavigationViewController.swift

```diff
@@ -304,6 +304,11 @@ public class NavigationViewController: UIViewController {
      */
     @objc public var annotatesSpokenInstructions = false
     
+    /**
+     A Boolean value that determines whether the dark style should be automatically activated when a route controller enters a tunnel.
+     */
+    @objc public var automaticallySwitchStyleForTunnels: Bool = false
```

Comment:
Nit: use the declarative form instead of the imperative form when naming a property. In Objective-C, a property’s getter can look identical to an action method:

```objc
// Does this line immediately switch the style?
[navigationViewController automaticallySwitchStyleForTunnels];
```

A clearer name would be `automaticallySwitchesStyleInsideTunnels` or `usesNightStyleInsideTunnels`.

Reply:
Honestly, I prefer to use `nightStyleInsideTunnelFeatureEnabled` to hint that variable is a feature flag. Just my `2¢`.